en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,812 @@
+# Integrating Services
+
+In this tutorial, you can learn how to integrate multiple services via Choreo. Here, let's consider a simple example where an online shoe store needs an application that captures online orders. For each order, the store also needs to generate a notification for the sales manager as well as update the inventory records.
+
+## Before you begin
+
+The following are required to try out this tutorial:
+
+- A Google account. In this Google account, open the Google Drive and create a Google sheet named `inventory` with the following table:
+
+    ![Google Sheet Extract](../assets/img/tutorials/inventory-service-google-sheet-extract.png)
+    | **item**    | **count** |
+    |-------------|-----------|
+    | trainers    | 100       |
+    | loafers     | 80        |
+    | kittenheels | 60        |
+    | maryjanes   | 40        |
+                            
+- A Twilio account
+
+## Step 1: Create a service to manage inventory records
+
+In this step, you are creating a service that consumes the orders captured by the `orders` service that you will be creating later, and then updates the inventory records based on the number of items ordered.
+
+To create this service, follow the procedure below:
+
+1. Access the Choreo Development Console and click **Services**. Then click **Create**.
+
+2. Enter `inventory` as the name and click **Create**.
+
+3. Click **PUT**, enter `/inventory` in the **Path** field, and then click **Save API**.
+
+    ![Resource Configuration](../assets/img/tutorials/inventory-service-api-trigger.png)
+
+4. Click **Variable**. Then enter information as follows:
```

Comment:
Lets remove the `responseString` variable

Reply:
Done